dataset.ttl

```diff
@@ -1224,188 +1292,437 @@ data:YarramanSubprovince
+    a geof:IgneousSubprovince , geo:Feature , sdo:Landform ;    
+    time:hasTime ischart:Visean , ischart:Pennsylvanian , ischart:Cisuralian , ischart:Guadalupian ;
+# provisionally placed with same age as KIP, as advised by Dave. Need to consider age range here instead #
+    sdo:name "Lakefield Subprovince"@en ;
+.
+
+data:PalumaSubprovince dct:isPartOf data:KENNEDY_IGNEOUS_PROVINCE ,
+    a geof:IgneousSubprovince , geo:Feature , sdo:Landform ;    
+    time:hasTime ischart:Visean , ischart:Mississippian , ischart:Pennsylvanian , ischart:Cisuralian ;
+    sdo:name "Paluma Subprovince"@en ;
+.
+
+data:TateSubprovince dct:isPartOf data:KENNEDY_IGNEOUS_PROVINCE 
+    a geof:IgneousSubprovince , geo:Feature , sdo:Landform ;    
+    time:hasTime ischart:Pennsylvanian , ischart:Cisuralian ;
+    sdo:name "Tate Subprovince"@en ;
+.
+
+data:YamboSubprovince dct:isPartOf data:ETHERIDGE_PROVINCE 
+    a geof:TectonisedMetomorphosedProvince , geo:Feature , sdo:Landform ;    
+    time:hasTime ischart:Calymmian , ischart:Stratherian ;
+    sdo:name "Yambo Subprovince"@en ;
+.
+data:Tasmanides dct:hasPart data:DELAMERIAN_OROGEN , LACHLAN_OROGEN , THOMSON_OROGEN , MOSSMAN_OROGEN , NEW_ENGLAND_OROGEN , dct:alternative TASMAN_OROGENIC_ZONE , TASMANIDES_ROCK_SYSTEM , TASMANIDE_SYSTEM , dct:description """The Tasmanides, part of the larger-scale Terra Australis Orogen that developed along the southern Panthalassan (Paleo-Pacific) margin of Gondwana, embraces five Cambrian to Triassic subduction-related orogens: the Delamerian, Lachlan, Thomson, Mossman, and the New England orogens (Foster & Gray, 2000; Rosenbaum, 2018)""" 
+# Delamerian & Lachlan not in Framework as poorly known in Queensland #
+        a geof:OrogenicCollage ,
+        geo:Feature ,
+        sdo:Landform ;
+    time:hasTime ischart:Triassic , ischart:Paleozoic ;
+    sdo:name "Tasmanides"@en .
+.
+
+data:UpperCretaceousCenozoicBasinTypes dct:hasPart data:AMBERLEY_BASIN , data:AUSTRAL_DOWNS_BASIN , data:BEAUDESERT_BASIN , data:BILOELA_BASIN , data:BOOVAL_BASIN , data:CAPRICORN_BASIN , data:CASUARINA_BASIN , data:DUARINGA_BASIN , data:EMERALD_BASIN , data:HERBERT_CREEK_BASIN , data:HILLSBOROUGH_BASIN , data:KALPOWAR_BASIN , data:KARUMBA_BASIN , data:LOWMEAD_GRABEN , data:MARION_BASIN , NAGOORIN_GRABEN , data:NORANSIDE_BASIN , data:OLD_CORK_BASIN , data:OXLEY_BASIN , data:PETRIE_BASIN , data:POMONA_BASIN , data:REDBANK_BASIN , data:ROSSMOYA_BASIN , data:SPRINGVLAE_BASIN , data:WATER_PARK_CREEK_BASIN , data:WOORABINDA_BASIN , data:YAMBA_BASIN , 
+        a geof:BasinType ,
+        geo:Feature ,
+        sdo:Landform ;
+    time:hasTime ischart:Quaternary , ischart:Neogene , ischart:Paleogene , ischart:UpperCretaceous ;
+    sdo:name "Upper Cretaceous � Cenozoic (largely Paleogene�Neogene) Basins"@en .

```

Comment:
are these symbols supposed to be hyphens here?

Reply:
Kate,
In the file, they are indeed hyphens. So, why are they showing up as �?